src/lib/components/Communities/CommunityHeader.js

```diff
@@ -32,8 +34,11 @@ class CommunityHeaderInfo extends React.Component {
     const logoLink = community ? community?.links?.logo : imagePlaceholderLink;
     return (
       <Container className="community-header">
-        <Image size="mini" src={logoLink} /> {community?.metadata?.title}
-        <CommunityHeaderButton />
+        <Image className="community-logo-header" src={logoLink} />
+        <div className='mt-10'>
+          {community?.metadata?.title}
```

Comment:
Creating a community I think requires a title. Can you check please and if it's the case then you should use `community.metadata.title` :)

Reply:
Right, it's a leftover. Thanks!